src/frontend/src/components/Posts/LoadAutoScroll.jsx

```diff
@@ -13,7 +13,7 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function LoadMoreButton({ onScroll }) {
+function LoadAutoScroll({ onClick }) {
```

Comment:
Other way around, this prop needs to be `onScroll` and below on the propTypes you need `onScroll` too.  You can see how it's called at https://github.com/Seneca-CDOT/telescope/blob/e98765297b55125d509166fe97a74eaa751cf3fd/src/frontend/src/components/Posts/Timeline.jsx#L48, which is sending an `onScroll` function.